apps/condo/domains/user/utils/tokens.js

```diff
@@ -3,23 +3,20 @@ const { v4: uuid } = require('uuid')
 
 const TOKEN_TYPES = {
     CONFIRM_PHONE: 'CONFIRM_PHONE',
-
     // CONFIRM_EMAIL: 'CONFIRM_EMAIL',
-    // SUDO: 'SUDO',
+    SUDO: 'SUDO',
 }
 
 const TOKEN_TYPE_BY_ABBREVIATION = {
     'cp': TOKEN_TYPES.CONFIRM_PHONE,
-
     // 'ce': TOKEN_TYPES.CONFIRM_EMAIL,
-    // 's': TOKEN_TYPES.SUDO,
+    's': TOKEN_TYPES.SUDO,
 }
 
 const ABBREVIATION_BY_TOKEN_TYPE = {
     'CONFIRM_PHONE': 'cp',
-
     // 'CONFIRM_EMAIL': 'ce',
-    // 'SUDO': 's',
+    'SUDO': 's',
```

Comment:
why do we need those abbreviations?

Reply:
1) We need to be able to determine token types by their appearance
2) I think it would be unnecessary to indicate the full name of the token in the token itself, the abbreviation is quite sufficient. This looks more concise

Reply:
idea: su?